spec/aws/rails/action_mailbox/requests/inbound_email_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'aws-sdk-s3'
+
+describe 'inbound email', type: :request do
+  let(:inbound_email_url) { '/rails/action_mailbox/amazon/inbound_emails' }
+
+  before do
+    stub_request(
+      :get,
+      'https://sns.eu-west-1.amazonaws.com/SimpleNotificationService-a86cb10b4e1f29c941702d737128f7b6.pem'
+    ).and_return(body: fixture_for(:certificate, type: :pem))
+  end
+
+  it 'receives inbound email' do
+    post inbound_email_url, params: JSON.parse(fixture_for(:inbound_email, type: :json)), as: :json
+
+    expect(response).to have_http_status(:no_content)
+    expect(ActionMailbox::InboundEmail.count).to eql 1
+  end
+
+  it 'receives an inbound email with data in s3' do
+    s3_email = fixture_for(:s3_email, type: :txt)
+
+    s3_client = Aws::S3::Client.new(stub_responses: true)
```

Comment:
the old way might have been causing issues in other specs, I don't know

Reply:
What was the old way being used here?

Reply:
Should we resolve this?

Reply:
Oh, it was like doing Aws[:s3] = blah blah. This way feels cleaner and maybe more inline with the other tests

Reply:
https://github.com/bobf/action_mailbox_amazon_ingress/blob/master/spec/requests/inbound_email_spec.rb#L23